### ISSUE TITLE
Add higher resolution of Notepad++'s standard icon set

### ISSUE THREAD

**Issue by deus-e-x** [notepad-plus-plus/notepad-plus-plus#9913]
I prefer to use Notepad++'s standard icon set for its look and color. However, there is no large set of standard icons available. So I would like to request to add a large set of the standard icons to Notepad++.

Checking through the contents of the Notepad++ binary I found, that the standard icon set is saved in the BMP format, while the other icon sets are saved in the ICO format, except for four PNG images. Wouldn’t it be preferable to have all icon sets be available in the PNG format to save space?

Related discussion on the Notepad++ user forum:
https://community.notepad-plus-plus.org/topic/21224/request-add-higher-resolution-of-notepad-s-standard-icon-set

**Comment by GOLEM777** [comment]
Yes, new icons are ugly!
I can create them, but I'm not sure if they will be in demand!

**Comment by thomassmith311** [comment]
The new fluent UI icons have a large version

**Comment by deus-e-x** [comment]
> The new fluent UI icons have a large version

Quote from my first post:
 > I prefer to use Notepad++'s standard icon set for its look and color.

**Comment by ArkadiuszMichalski** [comment]
Next request: https://community.notepad-plus-plus.org/topic/21706/large-toolbox-icons-in-color

**Comment by griggione** [comment]
Bonjour tous,

Dana Basel a écrit : 

> Small icons just don’t work with my old Boomer eyes.

**+1**

**Comment by stephane352** [comment]
I also really miss the standard large colour icons and would like to have them back !

**Comment by ArkadiuszMichalski** [comment]
Another ask:
https://community.notepad-plus-plus.org/topic/21730/new-release-notepad-v8-1-4/12?_=1630751084484

**Comment by sorganek** [comment]
I would like to have the option to re-install the standard large color icons in the toolbar.   I am fine to have the black and white icons as the default, but providing the option  for the color icons would be appreciated.

**Comment by DTG123** [comment]
I have 1440p monitor, I really need the larger icons, I'd prefer the colorized ones or some plugins for anything else but these Fluent junk icons.

**Comment by stephane352** [comment]
Any news on this ?  I also have a high resolution monitor and small icons are very difficult to see.
If it is not possible to have 6 choices of icons, would it be possible to have Standard Large rather than Standard Small in the list ?
My eyes are not getting younger, but given today's monitors resolution, I think most people would be fine with the bigger icons...
Thanks in advance.

**Comment by BoomerangAide** [comment]
Good to see there's already an issue about this.
I also wish there was a "large" version of the standard set, as the "fluent" things just don't look good to me.
But I'm not really asking for "higher resolution", a simple "**pixel resize**" with an integer multiplier would work for me, or even something that programmatically resize the images if that's available in the engine of the application, though in that case I would be worried about a possible blur.
Note: just like someone a bit above, I'm at 1440p, I wonder if it's an issue more for 2K and 4K screen owners?

**Comment by pryrt** [comment]
Another two users in the Community have 4k monitors and really want the standard icons in large format.

* https://community.notepad-plus-plus.org/topic/22831/icon-menu-are-very-small-in-4k

BTW, @LaTarsha , I am the "PeterJones" from the Community Forum; the post you linked had nothing to do with toolbar icons; it is solely the executable's icon (the icon you see in Windows Explorer for the Notepad++ application).

**Comment by drakkar20d** [comment]
I would also like the standard icons in a larger size , sorry but the "fluent" ones are really bad and I mean beyond woeful

**Comment by ChristopherSouter** [comment]
I also have a 1440p monitor, I'm 73 years old, ,y eyesight isn't so good anymore, and I, too, would very much like to see the return of the large standard icons.

**Comment by apopticon** [comment]
Larger standard icons would be VERY MUCH appreciated! 

With all the high res screens, and monster video cards, why all the black & white, flat UI's lately??? I would agree that skeuomorphism went a little too far in the past, but seriously - the "Fluent UI" icons just look like a bunch of incoherent hieroglyphics to me.

**Comment by BoomerangAide** [comment]
Suggestion for followers of this issue: add a 👍 to the first entry.
Maybe devs don't use the "sort by reaction" feature, but in case they do, it could help.

**Comment by donho** [comment]
There won't be the standard icon large set as internal option, since I don't have the large set.
OTOH, you could create your own set.

You can test this sample: [customizeToolbarIcons.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/8764714/customizeToolbarIcons.zip) with the instructions in this commit: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/d85b9a7464b681a3e9363b12666e90c4859edbef (by using [the latest built binaries of master](https://ci.appveyor.com/project/donho/notepad-plus-plus)).

**Comment by Hagsy62** [comment]
Re "There won't be the standard icon large set as internal option, since I don't have the large set.
OTOH, you could create your own set.", could somebody out there not send you the large set?  I'm also 60+ but still prefer the coloured large icons.  I don't understand any of your last line above - I just want to code my PHP.

Please, please can we have them back?

**Comment by alankilborn** [comment]
Like UDL and the upcoming themes collections -- supplied by users and not bundled with Notepad++ -- is a user collection of toolbar icons needed?

**Comment by pryrt** [comment]
@alankilborn ,
I am not sure that the toolbar icon sets have shown enough traction to create a Collection yet.

@Hagsy62 said,
> could somebody out there not send you the large set? I'm also 60+ but still prefer the coloured large icons. ... Please, please can we have them back?

The post you were replying to -- and quoted -- literally provided links to a full set of large icons for you to download.  Admittedly, the instructions Don gave were the old instructions -- but those were necessary on May 24th because that feature hadn't been released yet.  That feature has now been released, as of v8.4.2 from June 3, so you don't need to do fancy "latest build binaries", and can just use your normal Notepad++ installation, as long as you have v8.4.2 or newer.  

So to get the large icon set: Grab that zip file that @donho linked.  Then follow the [Toolbar Icon Customization instructions](https://npp-user-manual.org/docs/config-files/#toolbar-icon-customization-toolbaricons-xml) from the User Manual.  And after following those instructions, you will have the large icons.

**Comment by Hagsy62** [comment]
@pryrt , Thanks for taking the time to reply and for the advice.  However, (you knew that was coming) I'm using Windows 7, I've upgraded to 8.4.5 and my setup is like this (hopefully I'll be loading some images...) : 
![C Drive](https://user-images.githubusercontent.com/35226220/190500098-ed8bde36-9d52-4af5-956a-1347e448f3b4.jpg)
![nppLegacyIcons folder](https://user-images.githubusercontent.com/35226220/190500485-7b1d8724-b269-4fe8-a9ea-a0aea7add9ec.jpg)
and I've tried both of these versions of the XML file...
![XML File v01](https://user-images.githubusercontent.com/35226220/190501955-8e88e1a9-001b-4004-ac38-3192d540fcaa.jpg)
![XML File v02](https://user-images.githubusercontent.com/35226220/190501970-a61bb544-045e-4a9d-8e97-023c7baeb202.jpg)

Am I missing something?  Thanks for your patience :-)

**Comment by ArkadiuszMichalski** [comment]
@Hagsy62 You use installed NPP, so you should put `toolbarIcons.xml` in correct place (where `config.xml` file exist).

**Comment by pryrt** [comment]
>  you should put `toolbarIcons.xml` in correct place

Actually, both `toolbarIcons.xml` and the `toolbarIcons\` folder need to go in the same location -- so for @Hagsy62 , they will likely both go in `%AppData%\Notepad++` hierarchy.

(I did confirm that with a normal installation, if the XML is in AppData hierarchy but the folder is in program files hierarchy, it won't work; they both need to be in the AppData hierarchy)

**Comment by apopticon** [comment]
I think it comes down to whether it was "Installed for all users" which 
would be "Program Files" or "Installed for just you" which should be 
%appdata%



-------- Original Message --------
Subject: Re: [notepad-plus-plus/notepad-plus-plus] Add higher resolution 
of Notepad++'s standard icon set (#9913)
From: PeterCJ ***@***.***>
To: notepad-plus-plus/notepad-plus-plus 
***@***.***>
Date: 9/15/2022

**Comment by Hagsy62** [comment]
Hi all, Thanks for the advice again and after a long search for the "config.xml" file in my "appdata" folder, I found it at "Terry/AppData/Roaming/Notepad++/config.xml", "Terry" being the name of my PC.  Having found it, I've moved the "toolbarIcons.xml" file and the folder of the same name to my newly found location, restarted NP++ and Bingo!  Big old icons!
Thanks again for all the advice but if anybody else is using Windows 7, the above comments may be useful.

**Comment by Hagsy62** [comment]
PS  The precise location of that config file was "C:\Users\DELL\AppData\Roaming\Notepad++".

**Comment by alankilborn** [comment]
@Hagsy62 Your 2 recent comments would not be specific to Windows 7.  And the quickest way to the correct location is to open up Explorer and type `%appdata%\notepad++` into the address bar...it is even possible to do that in Notepad++'s _File_ > _Open_ > _File name_ box.

**Comment by Hagsy62** [comment]
Thanks @alankilborn , could you say which versions of Windows my comments **_would_** be specific to (to help anybody else reading this trying to get some help)?  And thanks for the "quickest way" to get to the correct location! I'm learning all the time thanks to you and everybody else here.  Thanks again, everyone :-)

**Comment by alankilborn** [comment]
@Hagsy62 said:

>could you say which versions of Windows my comments would be specific to

There is no answer to that.  The stuff you were talking about (config file paths/names for Notepad++) would be the same on any version of Windows.

**Comment by pryrt** [comment]
Moreover, _all_ of the advice since my earlier post was included in the link I had in my original reply -- they tell you to use `%AppData%\Notepad++` for a normal installation, or `<installfolder>\` for a portable or the settings directory from `-settingsDir` if you use that option.  (Admittedly, there was a mistake in the markdown which made the footnotes hard to follow correctly, though the information is there; that mistake was corrected a couple months back, but the updates to the manual haven't been published since then.)
